.github/workflows/e2e-test-llmisvc.yaml

```
@@ -173,9 +174,25 @@ jobs:
+          docker save ${{ env.VLLM_CPU_IMAGE }} -o /tmp/vllm-cpu.tar
+
+      - name: Upload vLLM CPU image artifact
+        uses: actions/upload-artifact@ea165f8d65b6e75b540449e92b4886f43607fa02 # v4.6.2
+        with:
+          name: vllm-cpu-image-${{ env.TAG }}
+          path: /tmp/vllm-cpu.tar
+          compression-level: 0
+          if-no-files-found: error
+
   test-llmisvc:
     runs-on: cncf-ubuntu-16-64-x86
```

actionlint (1.7.12)

[error] label "cncf-ubuntu-16-64-x86" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2025", "windows-2025-vs2026", "windows-2022", "windows-11-arm", "ubuntu-slim", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-24.04-arm", "ubuntu-22.04", "ubuntu-22.04-arm", "macos-latest", "macos-latest-xlarge", "macos-latest-large", "macos-26-intel", "macos-26-xlarge", "macos-26-large", "macos-26", "macos-15-intel", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xlarge", "macos-14-large", "macos-14", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)